data/vehicledata.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": [
```

Comment:
If you'd take a look at the previously added data. The schema is 

```
{
    "data": {
         ...data
    }
}

```

Please follow the same convention.